src/Compilers/CSharp/Test/Symbol/Symbols/CovariantReturnTests.cs

```diff
@@ -2,277 +2,930 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.CodeAnalysis.CSharp.Symbols;
 using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
 {
     public class CovariantReturnTests : CSharpTestBase
     {
+        private void VerifyOverride(CSharpCompilation comp, string methodName, string overriddenMethodName)
+        {
+            var method = comp.GlobalNamespace.GetMember(methodName);
+            var overridden = method.GetOverriddenMember();
+            Assert.Equal(overriddenMethodName, overridden.ToTestDisplayString());
```

Comment:
>Assert.Equal(overriddenMethodName, overridden.ToTestDisplayString()); [](start = 12, length = 69)

Consider also asserting the same for ```method```. Makes it easier to see what type did we assume for the overriding method, strengthens the tests.

Reply:
Did you mean adding a parameter so we can assert the display name of the method? The syntax we use for looking up a member is not compatible with the syntax generated for a display string.

---
In reply to: [416916024](https://github.com/dotnet/roslyn/pull/43673#discussion_r416916024) [](ancestors = 416916024)

Reply:
> Did you mean adding a parameter so we can assert the display name of the method?

Yes

---
In reply to: [416985050](https://github.com/dotnet/roslyn/pull/43673#discussion_r416985050) [](ancestors = 416985050,416916024)